src/Yasumi/Provider/CommonHolidays.php

```diff
@@ -551,4 +551,41 @@ public function winterTime(
 
         return null;
     }
+
+     /**
+     * (international) children's Day
+     *
+     * Children's Day is a day celebrated in over 145 states to honor children.
```

Comment:
It may be better to use the term "Country" rather than state. A state is often considered a subdivision.